src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -103,9 +104,6 @@
     line-height: 16px;
     text-align: left;
 }
-.tooltip__button--data > * {
-    opacity: 0.9;
-}
```

Comment:
This and the first change in the diff are meant to increase contrast and bring us closer to the figma look.